assimilation_code/programs/readme.rst

```diff
@@ -0,0 +1,195 @@
+   Subsets, combines, or alters observations from one or more observation sequence files 
+   and optionally writes them into a single output obs_seq file.
+
+`obs_loop/obs_loop <obs_loop/obs_loop.html>`_
+   A template to read in observations from one obs_seq file and write them,
+   optionally modified by user supplied code, to another obs_seq file.
+ 
+obs_utils/obs_sort
+   Do a complete sort of an obs_seq file by location, observation type, then variance.
+   An ancestor of obs_remove_dups.
+
+obs_utils/obs_remove_dups
+   Removes duplicate observations from an obs_seq file, which involves a complete sort
+   by time, location, observation type, then variance.
+ 
+`obs_common_subset/obs_common_subset <obs_common_subset/obs_common_subset.html>`_
+   Select the subset of observations, which were successfully assimilated, 
+   from two or more assimilation cases (which used the same obs_seq.out file).
+ 
+`obs_seq_verify/obs_seq_verify <obs_seq_verify/obs_seq_verify.html>`_
+   Reorders the observations from a forecast run of DART into a structure 
+   that is amenable for the evaluation of the forecast.
+ 
+
+obs_utils/obs_data_denial
+   THIS IS NOT YET DONE!
+   Help implement a data-denial experiment by randomly changing the error variance
+   of N of each obs type in an observation sequence file to a huge value.
+ 
+Assimilation Programs
+-----------------------------------
+ 
+`perfect_model_obs/perfect_model_obs <perfect_model_obs/perfect_model_obs.html>`_
+   Creates synthetic observation sequences from a hindcast model.
+ 
+`filter/filter <filter/filter.html>`_
+   Main Fortran program for driving ensemble filter assimilations.
+
+filter/filter.separate_seq
+   Like filter, but each task updates its own sequence in obs_space_diagnostics.
+   Included here just for future reference.
+
+`advance_time/advance_time <advance_time/advance_time.html>`_
+   Provides a shell-scripting-friendly way to increment and decrement calendar dates and times.
+ 
+`integrate_model/integrate_model <integrate_model/integrate_model.html>`_
+   Generic main program which advances a single ensemble member in ``perfect_model_obs`` 
+   or the serial or parallel version of the ``filter`` program.
+
+`integrate_model/integrate_model_parallel <integrate_model/integrate_model.html>`_
+   Generic main program which advances a single
+   ensemble member in ``perfect_model_obs`` or the serial ``
+   or parallel version of the ``filter`` program.``
+
+Evaluating Results
+-----------------------------------
+ 
+obs_diag 
+   Reads obs_seq.final files, calculates statistics, and writes them to NetCDF files 
+   for use by Matlab (or other) plotting scripts.
+   There are separate versions for models with different coordinate systems:
+   `1D <obs_diag/oned/obs_diag.html>`_,
+   `3D cartesian <obs_diag/threed_cartesian/obs_diag.html>`_,
+   `3D spherical <obs_diag/threed_sphere/obs_diag.html>`_, and 
+   3D spherical with streamflow.
```

Comment:
```suggestion
   There are separate versions for models with different coordinate systems:
   
   - `1D <obs_diag/oned/obs_diag.html>`_,
   - `3D cartesian <obs_diag/threed_cartesian/obs_diag.html>`_,
   - `3D spherical <obs_diag/threed_sphere/obs_diag.html>`_, and 
   - 3D spherical with streamflow.
```

or whatever formatting makes this into a list instead of putting them on the same line?  easier to see that way.

Reply:
done also capital Cartesian.